reporter/src/funTest/kotlin/TestData.kt

```diff
@@ -148,7 +148,10 @@ val ORT_RESULT = OrtResult(
                         binaryArtifact = RemoteArtifact.EMPTY,
                         sourceArtifact = RemoteArtifact(
                             url = "https://example.com/license-file-1.0.tgz",
-                            hash = Hash(value = "", algorithm = HashAlgorithm.SHA1)
+                            hash = Hash(
+                                value = "e824b43fc02fa979e1377164ba26a739790c281b",
```

Comment:
Maybe again highlight the fake hash by using zeroes?

Reply:
Done.